### ISSUE TITLE
No Edit input/outputs button

### ISSUE THREAD

**Issue by Shadowblitz16** [Zylann/godot_voxel#562]
**Describe the bug**
The documentation say this...
![image](https://github.com/Zylann/godot_voxel/assets/2453707/f4b44824-7c0b-45a1-a511-f8df84729497)
But there is no such button in the inspector...
![image](https://github.com/Zylann/godot_voxel/assets/2453707/1e88c432-c6de-4506-a0e4-e3ac687ad766)

**To Reproduce**
1. Create a voxel graph function
2. Add some inputs and outputs
3. Click on background
4. Goto inspector and see there is no Edit input/output option

[MineBound.zip](https://github.com/Zylann/godot_voxel/files/12789992/MineBound.zip)

I am also getting this in console...
```
 modules/voxel/editor/graph/voxel_graph_editor.cpp:928 - Graph compilation failed: The graph requires at least one input node matching 'offset' (CustomInput). Add the missing node or remove the input in I/O settings.
```

**Expected behavior**
Documentation should reflect how to expose graph function to other graphs 

**Screenshots**
I posted them above.

**Environment**
- OS: Kubuntu 22.02
- Graphics card:
![image](https://github.com/Zylann/godot_voxel/assets/2453707/0ba1bffc-bf7c-4119-85cb-d1818e2b6c72)
- Godot version:
![image](https://github.com/Zylann/godot_voxel/assets/2453707/1601e6f4-8b2d-4dd1-ada4-b6cad0525f7e)
- Module version:
![image](https://github.com/Zylann/godot_voxel/assets/2453707/68abc04b-1006-4528-968f-e5b9cada24d2)
- Renderer used: Forward+

**Comment by Zylann** [comment]
That step was removed recently, I forgot to update the docs. It was only allowing you to click "auto-generate" anyways, now it does it automatically when the graph compiles in the editor.

**Comment by Shadowblitz16** [comment]
> That step was removed recently, I forgot to update the docs. It was only allowing you to click "auto-generate" anyways, now it does it automatically when the graph compiles in the editor.
> 
> > modules/voxel/editor/graph/voxel_graph_editor.cpp:928 - Graph compilation failed: The graph requires at least one input node matching 'offset' (CustomInput). Add the missing node or remove the input in I/O settings.
> 
> I fixed something similar last week in [1396185](https://github.com/Zylann/godot_voxel/commit/139618535f6bbdd713c1ad3fbcc5e3801405d3f9), if your build is older than that, try a more recent version?

ok thankyou
